lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md

```diff
@@ -162,3 +162,30 @@ This example will show how to execute basic operations on lighty.io gNMI device
      ]
    }
    ```
+   
+   ## gNOI
+   Simulator implements the following gNOI gRPCs with implemented logic:
+   - file.proto:
+     - get - downloads dummy file
+     - stat - returns stats of file on path
+   - system.proto:
+     - time - returns current time
+   
+   Other implemented gNOI gRPCs but with no logic implemented (they return default response instance or some predefined static response)
+    are:
+   - file.proto:
+     - remove
+   - cert.proto:
+     - install
+     - getCertificates
+   - os.proto:
+     - install
+   - system.proto:
+     - reboot
+   - sonic.proto:
+     - showTechsupport
+     - copyConfig
+     - imageInstall
+     - imageRemove - returns error (used for testing purposes)
+     - imageDefault
```

Comment:
until they are supported & implemented, it is not needed to mention them here

Reply:
Done